src/mudlet.cpp

```diff
@@ -2079,16 +2079,16 @@ void mudlet::writeSettings()
     settings.setValue("maximized", isMaximized());
     settings.setValue("editorTextOptions", static_cast<int>(mEditorTextOptions) );
     settings.setValue("statusBarOptions", static_cast<int>(mStatusBarState) );
-    settings.setValue("reportMapIssuesToConsole", mIsToDisplayMapAuditErrorsToConsole );
+    settings.setValue("reportMapIssuesToConsole", mshowMapAuditErrors );
 }
 
 void mudlet::slot_show_connection_dialog()
 {
     auto pDlg = new dlgConnectionProfiles(this);
-    connect (pDlg, SIGNAL (signal_establish_connection( QString, int )), this, SLOT (slot_connection_dlg_finnished(QString, int)));
+    connect(pDlg, SIGNAL(signal_establish_connection(QString, int)), this, SLOT(slot_connection_dlg_finished(QString, int)));
```

Comment:
👍  That smelling error always bugged me - but not enough to do anything about it...